Resources/Textures/Interface/Actions/meta.json

```diff
@@ -54,6 +54,9 @@
         },
         {
             "name": "malfunction"
+        },
+        {
+            "name": "web"
```

Comment:
where did you took this from? if its from the same place linked in the copyright in this file disregard this but if it isnt you need to link it

Reply:
It is mine

Reply:
Then if you agree with the `license` at the top of the file you should include something like 

> web made by jackfrost (github) for SS14"

in the `copyright` field